scipy/spatial/distance.py

```diff
@@ -2113,7 +2115,8 @@ def pdist(X, metric='euclidean', *, out=None, **kwargs):
     2. ``Y = pdist(X, 'minkowski', p=2.)``
 
        Computes the distances using the Minkowski distance
-       :math:`||u-v||_p` (p-norm) where :math:`p \\geq 1`.
+       :math:`||u-v||_p` (:math:`p`-norm) where :math:`p > 0` (note
```

Comment:
Do we really need all this math notation here? We don't explain what p-norm is and specifically mentioning again in brackets that it is p-norm so it doesn't serve any purpose. So if we continue the sentence _[....] using Minkowski distance_ without the linebreaks

```suggestion
       in p-norm where p is a positive number (for values of p between 0 and 1 this constitutes only a quasi-metric). 
```

Reply:
In the next hunk you'll see that it used to be written like this already:
```
math:`||u-v||_p` (:math:`p`-norm) where :math:`p \\geq 1`.
```

I just tried to harmonise the two docstrings; I don't see the harm in a bit of math notation, but happy to change it.

Reply:
on the pages for cdist/pdist, I actually think it's a helpful visual aid - `||.||_p` is more easily recognizable than "Minkowski distance" (and not just because people mix up or forget the names).

Reply:
I'm coming to like the math-notation, to be honest. 0 < p < 1 is much more precise and immediately useful than "p between 0 and 1", where I need to spend some brain cycles about what happens at the boundaries)

Reply:
0 < p < 1 is also fine. But explaining notation with more math notation is just redundant in my opinion. People who don't know what p-norm is won't get it anyways and the ones who know won't care. Hence it's not doing any work. 

In case you want to keep it then the TeX needs `\| ... \|` for correct spacing by the way.

Reply:
> In case you want to keep it then the TeX needs `\| ... \|` for correct spacing by the way.

I did this now for all occurrences in that module.